src/theme/SearchBar/index.js

```diff
@@ -0,0 +1,266 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
I think we should leave the swizzled files out of the repo entirely. They aren't intended for use currently, and possibly never will be.

Reply:
✅